### DISCUSSION TITLE
Frankfurt server down?

### DISCUSSION THREAD

**Discussion by faragher** [markqvist/Reticulum#303] [General]
I've noticed the Frankfurt server is reporting as down for me. Nisa-node reports the same. Is this a known outage?

**Comment by markqvist** [comment]
Yes, the VM it was running on was a bit underprovisioned in terms of memory and disk space, and it was due for an overhaul. I'm not sure when I will be able to reinstall it and set it up again, so for now `dublin.connect.reticulum.network` is where the action is.

**Reply by faragher** [reply]
Well, the good news is it's a mesh network. 😄 

With Beleth, Nisa, and Between the Borders, (and anyone else I missed) there are a bunch of endpoints, so no rush getting it back up.

**Reply by kc1awv** [reply]
Beleth was an experiment between myself and a few others to get a field-expedient wireless network up and running, and now I believe that all the 'experimentation' with Beleth is done. Originally had it set up as a net within the net, but now it's just the one node attached to the testnet. Don't really need all the extra nodes now, but I've figured out how to use the rncp, rnx, and rnsh tools. I will do my best to keep Beleth up and running as a node for the community.

**Reply by faragher** [reply]
Thank you, it's a nice reference. Is that network status widget I see with Beleth and Nisa an example I keep overlooking, or was that custom-made?

Also, if you learned any neat lessons, I'm all ears. I mean to set up a system in a suitcase to handle a mesh network for incident command (gotta do something with that degree) and knowing what doesn't work would be extremely valuable. Long term. I'm literally porting Reticulum to C# just to get to testing the mobile nodes for the system. It'll be a while.

**Reply by kc1awv** [reply]
Taking lessons learned from the message board example in nomadnet, any .mu file can be a script. In my implementation, I'm using Python to display the output from `rnstatus`. Basically, it's just this:

```python
#!/usr/bin/python
import subprocess
rnstatus = subprocess.getoutput("/home/user/.local/bin/rnstatus")

print(rnstatus)
```

Then make the .mu file executable, reload nomadnet and browse to the page!

---

I'd be glad to start a separate discussion for "pop-up" Reticulum nodes in a proper location so we can all share how we do what we do :smile:

**Reply by faragher** [reply]
It would be useful, but right now all I can add is warnings about power supplies and poor shielding.
